homeassistant/components/group/strings.json

```diff
@@ -8,17 +8,20 @@
           "group_type": "Group type"
         }
       },
-      "cover": {
+      "binary_sensor": {
         "description": "Select group options",
```

Comment:
These descriptions don't add any value, let's remove them.

Reply:
Without those descriptions, the steps are displayed just like this:

![image](https://user-images.githubusercontent.com/14281572/157456520-7a1f8703-4fb2-4c8e-b926-34c63e621853.png)

![image](https://user-images.githubusercontent.com/14281572/157456599-81d9f5e5-b17f-4db8-8166-394f331a47ce.png)

I think it's a bit too minimal?

Reply:
Frontend should render the label for target and “all” needs to be more descriptive

Reply:
Right, it's now like this instead:
![image](https://user-images.githubusercontent.com/14281572/157633962-e54a7f02-ed9f-4f25-8739-3c162d1dbee4.png)